index.js

```diff
@@ -121,7 +121,14 @@ function filterDict(rules, excludedWords) {
  */
 function getRegExp(word) {
 	const wordPattern = escapeRegExp(word).replace(/'/g, "['’‘]");
-	return new RegExp(`(?:^|[^-\\w])(${wordPattern}(?= |\\. |\\.$|$))`, 'ig');
+	const punctuations = ['\\.', ',', ':', ';', '\\?', '!'];
+	const punctuationsRx = punctuations.map(
```

Comment:
`Rx` is quite obscure: 

```suggestion
	const punctuationsRegExp = punctuations.map(
```

Reply:
Done :+1: